src/Parsers/Access/ASTSettingsProfileElement.cpp

```diff
@@ -52,10 +52,22 @@ void ASTSettingsProfileElement::formatImpl(const FormatSettings & settings, Form
                       << applyVisitor(FieldVisitorToString{}, max_value);
     }
 
-    if (readonly)
+    switch (type)
     {
-        settings.ostr << (settings.hilite ? IAST::hilite_keyword : "") << (*readonly ? " READONLY" : " WRITABLE")
-                      << (settings.hilite ? IAST::hilite_none : "");
+        case SettingConstraintType::NONE:
+            break;
+        case SettingConstraintType::WRITABLE:
+            settings.ostr << (settings.hilite ? IAST::hilite_keyword : "") << " WRITABLE"
+                          << (settings.hilite ? IAST::hilite_none : "");
+            break;
+        case SettingConstraintType::CONST:
+            settings.ostr << (settings.hilite ? IAST::hilite_keyword : "") << " READONLY"
```

Comment:
We changed that name to `CONST`, didn't we? I think we should output only `CONST` but be able to parse both `CONST` and `READONLY`.

Reply:
Ok, lets fix it